resources/prometheus/prometheus-rules.yaml

```diff
@@ -476,3 +476,141 @@ spec:
             rhacs_org_id: "{{ $labels.rhacs_org_id }}"
             rhacs_cluster_name: "{{ $labels.rhacs_cluster_name }}"
             rhacs_environment: "{{ $labels.rhacs_environment }}"
+    - name: az-resources
+      rules:
+        - alert: WorkerNodesByAZMemoryQuotaOverCommit
+          expr: |
+            (
```

Comment:
I think these promQL queries a bit too lengthy. Could you try to break them down with recording rules?

Reply:
Maybe also keep in mind that we'd like to use some of these queries as part of our dashboard, so the shorter the better 😅